packages/node_modules/@ciscospark/redux-module-avatar/src/actions.js

```diff
@@ -0,0 +1,91 @@
+export const ADD_AVATAR = `avatar/ADD_AVATAR`;
+export const ADD_AVATAR_BEGIN = `avatar/ADD_AVATAR_BEGIN`;
+
+import {bufferToBlob} from '@ciscospark/react-component-utils';
+
+function addAvatar(userId, avatar) {
+  return {
+    type: ADD_AVATAR,
+    payload: {
+      userId,
+      avatar
+    }
+  };
+}
+
+function addAvatarBegin(userId) {
+  return {
+    type: ADD_AVATAR_BEGIN,
+    payload: {
+      userId
+    }
+  };
+}
+
+/**
+ * Fetches a group of users' avatars
+ * @param {Array} userIds
+ * @param {object} spark
+ * @returns {Thunk}
+ */
+export function fetchAvatarsForUsers(userIds, spark) {
+  const promises = [];
+  return (dispatch) => {
+    userIds.forEach((userId) => {
+      promises.push(dispatch(fetchAvatarForUserId(userId, spark)));
+    });
+    return Promise.all(promises);
+  };
+}
+
+/**
+ * Fetches an avatar for a given user id
+ * @param {string} userId
+ * @param {object} spark
+ * @returns {Thunk}
+ */
+export function fetchAvatarForUserId(userId, spark) {
```

Comment:
Do we still need this if we have `fetchSpaceAvatar`? Seems like all the logic is repeated. We could have something like a single `fetchAvatar({userIds: [], space}, sparkInstance)`